tests/Bridge/Symfony/CocurSlugifyExtensionTest.php

```diff
@@ -38,36 +40,37 @@ protected function setUp()
      */
     public function testLoad()
     {
-        $twigDefinition = m::mock('Symfony\Component\DependencyInjection\Definition');
+        $twigDefinition = $this->createMock(Definition::class);
         $twigDefinition
-            ->shouldReceive('addTag')
-            ->with('twig.extension')
-            ->once()
-            ->andReturn($twigDefinition);
+            ->expects($this->once())
+            ->method('addTag')
+            ->with($this->equalTo('twig.extension'))
+            ->willReturn($twigDefinition);
         $twigDefinition
-            ->shouldReceive('setPublic')
-            ->with(false)
-            ->once();
+            ->expects($this->once())
+            ->method('setPublic')
+            ->with($this->equalTo(false))
+            ->willReturn($twigDefinition);
 
-        $container = m::mock('Symfony\Component\DependencyInjection\ContainerBuilder');
+        $container = $this->createMock(ContainerBuilder::class);
         $container
-            ->shouldReceive('setDefinition')
-            ->with('cocur_slugify', m::type('Symfony\Component\DependencyInjection\Definition'))
-            ->once();
+            ->expects($this->exactly(2))
+            ->method('setDefinition')
+            ->withConsecutive(
+                [$this->equalTo('cocur_slugify'), $this->isInstanceOf(Definition::class)],
+                [$this->equalTo('cocur_slugify.twig.slugify', $this->isInstanceOf(Definition::class))]
```

Comment:
Here is a mis-placed parentheses
```suggestion
                [$this->equalTo('cocur_slugify.twig.slugify'), $this->isInstanceOf(Definition::class)]
```